lib/standalone-single-spa.js

```diff
@@ -140,6 +140,30 @@ class StandaloneSingleSpaPlugin {
       attributes: { type: "systemjs-importmap" },
       innerHTML: JSON.stringify(importMap, null, 2),
     });
+    scripts.push({
+      tagName: "meta",
+      voidTag: true,
+      attributes: {
+        name: "importmap-type",
+        content: "systemjs-importmap",
+      },
+    });
+    scripts.push({
+      tagName: "script",
+      voidTag: false,
+      attributes: {
+        defer: false,
+        src:
+          "https://cdn.jsdelivr.net/npm/import-map-overrides/dist/import-map-overrides.js",
+      },
+    });
+    scripts.push({
+      tagName: "import-map-overrides-full",
+      voidTag: false,
+      attributes: {
+        "dev-libs": true,
+      },
```

Comment:
My only feedback here is that this'll need to surface some of the import-map-overrides configuration options if an org doesn't use the defaults. The only cases of real significance would be if a different `show-when-local-storage` is set in their root config and the org wants to do the same in standalone. _Probably_ wouldn't be need for a long while (until someone asks for that) so it could be left for a future implementation.

Reply:
In standalone mode, I think most of the time we can just show the import-map-overrides UI without checking for a local storage flag. Which is what I set up here. But I agree we should let people turn off import-map-overrides entirely, or specify the local storage flag to check. I'll add that here.